From 8b65555a42f1adb2c00f71fb36e51f2e2b6d7c69 Mon Sep 17 00:00:00 2001
From: Divya Theja <quic_divyathe@quicinc.com>
Date: Tue, 11 Jan 2022 18:28:38 +0530
Subject: [PATCH] gstomxh265enc: Add OMX_VIDEO_HEVCHighTierLevel1 case

Add case OMX_VIDEO_HEVCHighTierLevel1 in switch flow and return
the caps according to the profile and level returned by OMX structure

Signed-off-by: Divya Theja <quic_divyathe@quicinc.com>
---
 omx/gstomxh265enc.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/omx/gstomxh265enc.c b/omx/gstomxh265enc.c
index d538374..a008819 100644
--- a/omx/gstomxh265enc.c
+++ b/omx/gstomxh265enc.c
@@ -697,6 +697,9 @@ gst_omx_h265_enc_get_caps (GstOMXVideoEnc * enc, GstOMXPort * port,
         tier = "main";
         level = "6.2";
         break;
+      case OMX_VIDEO_HEVCHighTierLevel1:
+        tier = "high";
+        level = "2";
       case OMX_VIDEO_HEVCHighTierLevel4:
         tier = "high";
         level = "4";
-- 
2.17.1

